packages/flame/lib/src/components/position_component.dart

```diff
@@ -98,6 +99,10 @@ class PositionComponent extends Component
   final NotifyingVector2 _size;
   Anchor _anchor;
 
+  /// The angle where this component is looking at in its
+  /// default state, i.e. when [angle] is equal to zero.
+  double nativeAngle;
```

Comment:
This is still a bit unclear: if nativeAngle is 0, then what does it mean? If I want to have my component pointing up, or to the right, then what values should I use?

Reply:
Modified the comments to explain it better with meaning of common ortho angles